imap-proto/src/parser/rfc3501/mod.rs

```diff
@@ -60,15 +64,15 @@ named!(mailbox<&str>, map!(
 
 named!(flag_extension<&str>, map_res!(
     recognize!(pair!(tag!("\\"), take_while!(atom_char))),
-    str::from_utf8
+    std::str::from_utf8
```

Comment:
Also these changes to add `std` before `str` belong in the other commit if we want them at all.